IM.I31KZ_SC3_formatted.xml

```diff
@@ -0,0 +1,792 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<seiscomp xmlns="http://geofon.gfz-potsdam.de/ns/seiscomp3-schema/0.8" version="0.8">
```

Comment:
This file needs to be stripped down significantly. It should only contain 2-3 stations with 1-3 channels each, so that the size is reduced as much as possible.

Reply:
OK. File can be found in `tests/data/IM.I31KZ.sc3ml`

Reply:
OK.